boxwise_flask/models.py

```diff
@@ -24,5 +25,54 @@ def __unicode__(self):
 
     @staticmethod
     def get_camps():
-        # SELECT distinct id, organisation_id FROM dropapp_dev.camps;
         return Camps.select().order_by(Camps.name)
+
+
+class Cms_Usergroups_Camps(db.Model):
+    camp_id = CharField()
+    cms_usergroups_id = CharField()
+
+    class Meta:
+        primary_key = CompositeKey('camp_id', 'cms_usergroups_id')
+
+    def __unicode__(self):
+        return self.name
+
+    @staticmethod
+    def get_camp(usergroup_id):
+        camp = (Cms_Usergroups_Camps
+            .select(Cms_Usergroups_Camps.camp_id)
+            .where(Cms_Usergroups_Camps.cms_usergroups_id == usergroup_id)
+            )
+        return camp
+
+
+class Cms_Users(db.Model):
+    id = CharField()
+    organisation_id = CharField()
+    naam = CharField()
+    name = CharField(column_name='naam')
+    email = CharField()
+    cms_usergroups_id = CharField()
+    valid_firstday = CharField()
+    valid_lastday = CharField()
+
+    def __unicode__(self):
+        return self.name
+
+    @staticmethod
+    def get_all_users():
+        return Cms_Users.select().order_by(Cms_Users.naam)
+
+    @staticmethod
+    def get_user(email):
+        q = Cms_Users.select().where(Cms_Users.email == email).get()
+        camps = Cms_Usergroups_Camps.get_camp(q.cms_usergroups_id)
+        listOfDicts = list(camps.dicts())
+        listOfCamps = []
+        for item in listOfDicts:
+            listOfCamps.append(item['camp_id'])
+
+        q.camp_id = listOfCamps
```

Comment:
Since `get_camp` return a list of camp IDs, assigning `list(camps)` might be sufficient?

Reply:
I had to do a bit more runaround... get_camps gives you a modelSelect, not a python list, so you have to gonvert to dictionaries first, and pull out only the camp_id part of the dictionary